sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Removing inactive session when it has timed out, so `ReactorConnection.getSession(String)` does not return the same session.
```

Comment:
Change `Removing`  to  `Removed` ?

If there is a correctponding issue,  we can also attached.

Reply:
Thanks for the catch. There was no GitHub issue associated with it. :( I should create one though so we can redirect people if they see the same issue.